### PR TITLE
OCPBUGS-18412:updates command for changing log levels

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OCPBUGS-18412

Link to docs preview:
https://68168--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources#nw-ovn-kubernetes-change-log-levels_ovn-kubernetes-sources-of-troubleshooting-information:~:text=ovnkube%2Dmaster%20components.-,Apply,-the%20ConfigMap%20file

QE review:
- [x] QE has approved this change.

Additional information:
PTAL @rbbratta

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Review by rbbratta — APPROVED** [review]

File: modules/nw-ovn-kubernetes-change-log-levels.adoc [hunk 1]
```diff
@@ -72,7 +72,7 @@ data:
 +
 [source,terminal]
 ----
-$ oc create configmap env-overrides.yaml -n openshift-ovn-kubernetes
+$ oc apply -f configmap env-overrides.yaml -n openshift-ovn-kubernetes
```
Comment (rbbratta) [inline modules/nw-ovn-kubernetes-change-log-levels.adoc:75]: no `configmap`

**Review by rbbratta — CHANGES_REQUESTED** [review]

File: modules/nw-ovn-kubernetes-change-log-levels.adoc (hunk 1 above)
Comment (JoeAldinger) [reply modules/nw-ovn-kubernetes-change-log-levels.adoc]: @rbbratta look good now?

**Review by rbbratta — APPROVED** [review]
/lgtm

**Pull request merged**